### ISSUE TITLE
Add 'keycloak-authorization' integration test based on Wiremock

### ISSUE THREAD

**Issue by sberyozkin** [quarkusio/quarkus#16276]
## Description
Testing various `keycloak-authorization` configuration options, including the caching of the responses, would help to increase the coverage and also show the users how they can quickly test it without having to set up Keycloak 

@cemnura Hi - this is another issue we can look at as part of improving the test support for OIDC/Keycloak :-)

**Comment by cemnura** [comment]
@sberyozkin would love to take a look 👍

**Comment by tomrist9** [comment]
Hi,

I would like to work on this issue as my first contribution to Quarkus.

I saw the previous WIP PR and the later discussion suggesting that any Keycloak Authorization test coverage should probably be based on the existing WireMock dev service, and extended if needed.

Could you please confirm whether this issue is still relevant and whether starting with a small missing Keycloak Authorization scenario using the current WireMock test support would be a good approach?

Thanks!

**Comment by michalvavrik** [comment]
I don't think we should be implementing this issue, we have barely any feedback regarding Keycloak Authorization. We don't need more tests or a new WireMock dev service. We plan to implement [Authorization API from OpenId AuthZen](https://github.com/quarkusio/quarkus/issues/52233) and in the future, we could deprecate this extension. I'd just close this issue, but it is @sberyozkin call.
